doc/en/doctest.rst

```diff
@@ -130,3 +130,20 @@ which can then be used in your doctests directly::
         10
         """
         pass
+
+
+Output format
+-------------
+
```

Comment:
Sorry realized one last thing: please add a `.. versionadded:: 3.0` here to make it clear this option was introduced in that version.